### ISSUE TITLE
Screen Reader issue

### ISSUE THREAD

**Issue by sergeicodes** [shower/shower#323]
## Screen Reader issue

When actively navigating through slide elements, screen reader reads the content from both the current slide and hidden `<section role="region">`.

### Steps to reproduce

1. Create 2 slides as follows

```html
<section class="slide">
  <h2>Slide one</h2>
</section>

<section class="slide">
  <h2>Slide two</h2>
</section>
```

2. Open 1st slide in full screen
3. Go to 2nd slide
4. Start a screen reader (_VoiceOver_ on this example)
5. Open _Rotor_ panel
6. See the _Heading list_

### Expected

`<h2>` element to appear once.

### Actual

`<h2>` element appears twice - 1st from the actual slide, 2nd from `<section role="region">` (see screenshot).

![shower-screen-reader-bug](https://user-images.githubusercontent.com/10166916/40684994-483b415a-6393-11e8-9e4b-e04e83f1eace.gif)

**Comment by pepelsbey** [comment]
Thank you! Any idea for a possible fix?

**Comment by krigersergei** [comment]
I would get rid of a hidden _a11y_ element and use a real slide for updating screen reader. Will try and see how it works.

**Comment by pepelsbey** [comment]
Well, there’s a problem with that approach: currently Shower doesn’t focus slides while you navigating them, it just switches class names. In this case live region is the only option. But we can change Shower’s core behaviour to actually make slides focusable (`tabindex="0"`, etc.) In this case every slide will be announced as a section labeled with `<h2>`. But it requiers some coding and testing.

Would you like to become a maintainer of a11y side of the Shower project? I’ll be glad to add you to organization and feature you everywhere. It don’t think it will take too much time, apart from implementing this feature.

**Comment by krigersergei** [comment]
Yeah, `tabindex` is the easiest way. Maybe not the best, but should work.

Sure, you can add me as a maintainer and I will check it when I have some more time.
